active/NavigationView/BottomItems/BottomMenuItems2.md

```diff
@@ -0,0 +1,194 @@
+See the left-aligned NavigationView example below for reference:
+
+![NavigationView with labels](./leftnav-anatomy2.png)
+
+NavigationView provides a built-in 
+[Settings item](https://docs.microsoft.com/en-us/uwp/api/microsoft.ui.xaml.controls.navigationview.settingsitem?view=winui-2.3), 
+which is always positioned at the bottom or right of the Pane, and a 
+[PaneFooter](https://docs.microsoft.com/en-us/uwp/api/microsoft.ui.xaml.controls.navigationview.panefooter?view=winui-2.3) 
+area that positions custom content adjacent to the Settings item.
+Although an app developer can place any content into PaneFooter, including other NavigationViewItems, 
+this content will not participate in the selection model and animations of 
+the MenuItems navigation list and Settings item. So if an item in MenuItems is selected, and the user
+clicks on an item in the PaneFooter, it will not de-select the preview item.
+
+_Open question:_ Should PaneFooter be deprecated because this new API is a direct replacement?
+
+Some apps may wish to visually group navigation items into top-aligned and bottom-aligned lists
+(or left- and right-aligned lists), for example:
+
+![NavigationView in Left display mode showing FooterMenuItems](FooterItems_Left.png)
+
+![NavigationView in Top display mode showing FooterMenuItems](FooterItems_Top.png)
+
+While this grouping is possible today by putting NavigationViewItems into the PaneFooter, 
+the approach creates a poor selection and interaction experience as described above.
+That's being corrected in this spec with a new FooterMenuItems property.
+
+As a result, keyboard and assistive technology users will feel that the footer list behavior meets
+their expectations -  the top and bottom menu items will be specified as two separate lists, 
+but it will be smooth to navigate from one to the other. See more in Accessibility section below.
+Selection behavior and animations will also feel better integrated into the whole NavigationView control, 
+as bottom menu items will participate in the same selection model as top menu items. 
+
+# Description
+
+## NavigationView.FooterMenuItems
+
+Use the FooterMenuItems to place navigation items at the end of the navigation pane,
+contrasted with the MenuItems property which places items at the start of the pane.
+
+```cs
+public IList<object> FooterMenuItems { get; }
+```
+
+The default Settings item will be included as the last item in the FooterMenuItems collection. 
+As a result, adding items into FooterMenuItems will by default display them before the Settings item. 
+The Settings item can still be toggled using the `IsSettingsVisible` property. 
+Note: this is the recommended way to remove the Settings item, although it's possible 
+to remove it manually by removing it from the FooterMenuItems collection itself. 
+
+Below is an example of a NavigationView with a FooterMenu where the Mail item is selected.
+
+![NavigationView with FooterMenu](footer-menu-mockup.png)
```

Comment:
This doesn't seem realistic. "Play", "Save", "Refresh" and "Download" aren't things you navigate to. They are immediate actions.

A more realistic set of items might be

(Top)
* Catalog
* On loan
* On hold
* Wish lists
* Your reviews
* Recommendations
* Challenges

(bottom)
* History
* Change account (non-navigation element)
* Settings